### ISSUE TITLE
Ensures that lists are structured correctly (.list-disc.list-inside)

### ISSUE THREAD

**Issue by Govardhani-T** [microsoft/onnxruntime#21639]
### GitHub Tags:
#A11ySev2; #A11yTCS; #DesktopWeb; #SH_ONNX Runtime & Ecosystem_Web_Mar2024; #ONNX Runtime & Ecosystem; #WCP; #Win11; #FTP; #P3_WCP; #ONNX Runtime; #ChromiumEdge; A11yWCAG2.1; #A11yMAS; #list; #WCAG1.3.1; #A11yAuto #info and relationship; #Narrator
**"Please do not close this bug. This bug should only be closed by TCS, C+AI Accessibility or the Divisional Driver".**
### Environment Details:
Application Name: WCP URLs - ONNX Runtime
URL: https://onnxruntime.ai/training
Edge version: 123.0.2420.53 (Official build) (64-bit)
OS: Windows 11 version 23H2
### Steps to Reproduce
1. Hit the URL "https://onnxruntime.ai/training".
2. Run Accessibility Insights for web.
3. Observe that Ensures that lists are structured correctly (.list-disc.list-inside)
### Issue: 
Ensures that lists are structured correctly (list - https://accessibilityinsights.io/info-examples/web/list)
### Target application: 
ONNX Runtime | Training - https://onnxruntime.ai/training
### Element path: 
div[data-svelte-h="svelte-n4v4jx"] > .list-disc.list-inside
### Snippet: 
<ul class="list-disc list-inside">Examples: <li>Image / Audio classification</li> <li>Text Prediction</li></ul>
### How to fix: 
Fix all of the following:
  List element has direct children that are not allowed: #text

Environment: Microsoft Edge version 127.0.0.0

====

This accessibility issue was found using Accessibility Insights for Web 2.44.0 (axe-core 4.9.1), a tool that helps find and fix accessibility issues. Get more information & download this tool at http://aka.ms/AccessibilityInsights.
### Attachment:
![image](https://github.com/user-attachments/assets/ee97284b-1837-4172-9f61-0ed33f7e8843)
![image](https://github.com/user-attachments/assets/c5c9e8cd-3f03-4e81-b1a8-7456d52e9fa9)

**Comment by Samhitha66** [comment]
#Rev:samhithaks;

**Comment by MaanavD** [comment]
Hey @Govardhani-T this should be addressed :)

**Comment by PadminiGariki** [comment]
#Closed; #A11yFixed; #Regressed:10-07-25;
verified the issue in below environment.
URL: https://onnxruntime.ai/training".

Issue is fixed. Hence the bug is closed.
For reference, attachment is added below.

<img width="1365" height="680" alt="Image" src="https://github.com/user-attachments/assets/51cde384-9e5d-47e5-9816-61ae9b518063" />
